src/main/java/org/gradle/profiler/ScenarioLoader.java

```diff
@@ -50,6 +50,7 @@
     private static final String BUCK = "buck";
     private static final String MAVEN = "maven";
     private static final String WARM_UP_COUNT = "warm-ups";
+    private static final String BUILD_COUNT = "builds";
```

Comment:
```suggestion
    private static final String ITERATION_COUNT = "iterations";
```

The command line option is `--iterations`, so I would use the same name in the scenario file. I don't know why the command line option is `--warmups` while the scenario file option ended up being `warm-ups`.

Reply:
Fixed